tracer/src/Datadog.Trace.ClrProfiler.Native/clr_helpers.h

```diff
@@ -467,6 +469,45 @@ struct FunctionMethodSignature
     }
 };
 
+struct FunctionLocalSignature
+{
+private:
+    PCCOR_SIGNATURE pbBase;
+    unsigned len;
+    ULONG numberOfLocals = 0;
+    std::vector<TypeSignature> locals;
+
+public:
+    FunctionLocalSignature() : pbBase(nullptr), len(0)
+    {
+    }
+    FunctionLocalSignature(PCCOR_SIGNATURE pb, unsigned cbBuffer, std::vector<TypeSignature>&& localsSigs) :
+        pbBase(pb), len(cbBuffer), numberOfLocals(static_cast<ULONG>(localsSigs.size())), locals(std::move(localsSigs))
+    {
+    }
+    ULONG NumberOfLocals() const
+    {
+        return numberOfLocals;
+    }
+    WSTRING str() const
+    {
+        return HexStr(pbBase, len);
+    }
+    std::vector<TypeSignature> GetMethodLocals() const
+    {
+        return locals;
```

Comment:
is it on purpose to copy the whole vector at each call to `GetMethodLocals` ?
(depends on how many times it's called and how many elements in the vector)

Reply:
That's a good point. I'll return it as const reference to gain some perf. Thanks!

Reply:
Done in 07de2ba1da178bb90d9a5111327945f614c4eeee.